task-forces/wcag2ict/work-statement.php

```diff
@@ -10,75 +10,77 @@
 		<section id="status">
 			<h2>Status</h2>
-			<p>This task force is being reopened to update WCAG2ICT for WCAG 2.x. This is a draft work statement.</p>
+			<p>This task force is being reopened to update WCAG2ICT for WCAG 2.x. The TF will pursue continuation of the work in the upcoming AGWG Charter. This is a draft work statement.</p>
 		</section>
 		<section id="objective">
 			<h2>Objective</h2>
-			<p>The objective of theWCAG2ICT Task Force is to develop documentation describing how WCAG 2.1 and 2.2 and its principles, guidelines, and success criteria could apply to non-Web Information and Communications Technologies (ICT). </p>
-			<p>This Task Force was formed to update the existing document: “Guidance on Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)” working group Note. Updates are needed to include new guidelines, principles and success criteria added in the WCAG 2.1 Recommendation as well as WCAG 2.2 will be a Recommendation before the updated Note is published. WCAG2ICT was developed to describe how to apply WCAG 2.0 and its principles, guidelines, and success criteria to non-web ICT (Information and Communications Technology) that are not web content.WCAG2ICT also plans to add new content that identifies problematic WCAG provisions when applied to non-Web ICT.</p>
+			<p>The objective of the WCAG2ICT Task Force is to develop documentation describing how WCAG 2.x and its principles, guidelines, and success criteria could apply to non-Web Information and Communications Technologies (ICT). </p>
+			<p>This Task Force was formed to update the 2013 “<a href="https://www.w3.org/TR/wcag2ict/">Guidance on Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)</a>” working group Note published on 5 September 2013. Updates are needed to include new guidelines and success criteria added in the WCAG 2.1 Recommendation as well as new requirements added in WCAG 2.2. WCAG2ICT was developed to describe how to apply WCAG 2.0 and its principles, guidelines, and success criteria to non-web ICT (Information and Communications Technology) that are not web content. WCAG2ICT also plans to add new content that identifies problematic WCAG provisions when applied to non-Web ICT.</p>
 		</section>
 		<section id="scope">
 			<h2>Scope of Work</h2>
 			<p>The work of the task force includes: </p>
 		</section>
 		<ul>
-			<li>Determining how to apply each of the new WCAG 2.1 and WCAG 2.2 (hereafter called “WCAG 2.x”) guidelines and A, AA, and AAA success criteria to non-Web ICT; (this includes provision of additional details for application to closed product software; and</li>
+			<li>Determining how to apply each of the new WCAG 2.1 and WCAG 2.2 (hereafter called “WCAG 2.x”) guidelines and A and AA success criteria to non-Web ICT; (this includes provision of additional details for application to closed product software and where applying individual criteria might be difficult and why; and</li>
```

Comment:
There is no closing parenthesis matching the starting parenthesis before the word "this".

Reply:
@cwadamsoforacle Removed extraneous parenthesis.